openpype/modules/kitsu/plugins/publish/integrate_kitsu_review.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import gazu
+import pyblish.api
+
+
+class IntegrateKitsuReview(pyblish.api.InstancePlugin):
+    """Integrate Kitsu Review"""
+
+    order = pyblish.api.IntegratorOrder + 0.01
+    label = "Kitsu Review"
+    # families = ["kitsu"]
+    optional = True
+
+    def process(self, instance):
+
+        context = instance.context
+        task = context.data["kitsu_task"]
+        comment = context.data["kitsu_comment"]
+
+        for representation in instance.data.get("representations", []):
+
+            local_path = representation.get("published_path")
```

Comment:
Is `local_path` a movie file path? If yes, could you please explicit the name?

Reply:
It is the review file path.